.github/workflows/release.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - dev
```

Comment:
This will allow us to run `npm run changeset` in the `dev` branch, and then run the `Publish dev version` Github Action to publish the new versions of packages.

If you need to publish packages for testing, merge the code into `dev` first, instead of `main`.
@imstar15